src/components/PlayerBoard/Info.vue

```diff
@@ -50,7 +50,7 @@
         <g transform="translate(16, 1)" v-b-tooltip title="Satellites">
           <image xlink:href='../../assets/resources/satellite.svg' :height=155/211*22 width=22 x=-11 y=-8
           transform="scale(0.07)" />
-          <text :class="['board-text']" transform="translate(1,0) scale(0.7)">{{ data.satellites }}</text>
+          <text :class="['board-text']" transform="translate(1,0) scale(0.7)">{{ data.effectiveSatellites }}</text>
```

Comment:
After seeing your commits, I realize that `player.eventConditionCount('sat')` could have been used here instead of `data.satellites + data.buildings.sp`.